modules/api-server/src/socket-io/index.ts

```diff
@@ -0,0 +1,130 @@
+import { Server, Socket } from 'socket.io';
+import { roomService } from 'services/room';
+import { Room } from 'db/models/room';
+
+interface CustomSocket extends Socket {
+  roomId?: number;
+  userId?: number;
+}
+
+export function setupSocketEvents(io: Server): void {
+  io.on('connection', (socket: CustomSocket) => {
+    console.log(`A user ${socket.id} connected`);
+
+    let iterations = 0;
+    let currentTeam = 'proTeam';
+
+    function startPreparationTimer(room: Room) {
+      console.log(`Preparation is started ${currentTeam}, ${room.id}`);
```

Comment:
Try to get rid of console.log that may be called often in production, as it sometimes can affect costs of logs highly(if it should produce lot of logs).
Not a critical for current stage.